.gitlab/corona-jobs.yml

```diff
@@ -11,5 +11,5 @@ rocm_4_5_2_fortran_clang_13_0_0 (build and test on corona):
 
 rocm_4_5_2_fortran_clang_13_0_0(build and test on corona):
```

Comment:
I think it will be wise to rename one of those job so that they are easier to differentiate.

Reply:
I was thinking the same thing!